### PR TITLE
Emphasis: basic text formatting

### PR DESCRIPTION
**Pull request by Sudrien**
Should I mention XEP-0071 somewhere?

### CODE REVIEW

File: emphesis/candy.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+			{ plain: '&lt;u&gt;underline&lt;/u&gt;',			regex: /(\&lt;u\&gt;(.*?)\&lt;\/u\&gt;)/igm,				plain: "+$2+",		xhtml: "<span style='text-decoration: underline;'>$2</span>" },
+			{ plain: '&lt;ins&gt;underline&lt;/ins&gt;',		regex: /(\&lt;ins\&gt;(.*?)\&lt;\/ins\&gt;)/igm,			plain: "+$2+",		xhtml: "<span style='text-decoration: underline;'>$2</span>" }
+			]
+
+	self.init = function() {
+
+	
+		$(Candy).on('candy:view.message.before-send', function(e, args) {
+
+			var working_plain_message = args.message;
```
Comment (mweibel) [inline emphesis/candy.js]: Could you adhere to the standard code style of Candy? This means: camelCase Variables, Correct indenting

**Comment by mweibel** [comment]
Thanks for your PR. Will merge as soon as I'm happy with the code style.

**Comment by bklang** [comment]
I don't mean to nitpick, but the correct spelling is "emphasis" ('a' instead of 'e'). Was this an intentional misspelling?

File: emphesis/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+```HTML
+<script type="text/javascript" src="candyshop/emphesis/candy.js"></script>
+```
+
+Call its `init()` method after Candy has been initialized:
+
+```javascript
+Candy.init('/http-bind/', {});
+
+// enable /me handling
```
Comment (melissanoelle) [inline emphesis/README.md]: This comment seems to be non-specific to be inaccurate for this plugin.

**Comment by melissanoelle** [comment]
You could mention XEP-0071 in the README.md perhaps?

And also curious as to the spelling nit on "Emphesis" vs. "Emphasis".

**Comment by Sudrien** [comment]
@bklang, @melissanoelle - I thought the spelling was a dictionary-approved variation. I was incorrect.

**Comment by melissanoelle** [comment]
:+1:

File: emphasis/candy.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		{ plain: '[s]strikethrough[/s]', regex: /(\[s\](.*?)\[\/s\])/igm, plain: "-$2-", xhtml: "<span style='text-decoration: line-through;'>$2</span>" },
+		{ plain: '&lt;s&gt;strikethrough&lt;/s&gt;', regex: /(\&lt;s\&gt;(.*?)\&lt;\/s\&gt;)/igm, plain: "-$2-", xhtml: "<span style='text-decoration: line-through;'>$2</span>" },
+		{ plain: '&lt;del&gt;strikethrough&lt;/del&gt;', regex: /(\&lt;del\&gt;(.*?)\&lt;\/del\&gt;)/igm, plain: "-$2-", xhtml: "<span style='text-decoration: line-through;'>$2</span>" },
+		{ plain: '+underline+', regex: /((^|\s|\>)\+(.*?)\+(\s|\<|$))/gm, plain: "$2+$3+$4", xhtml: "$2<span style='text-decoration: underline;'>$3</span>$4" },
+		{ plain: '[u]underline[/u]', regex: /(\[u\](.*?)\[\/u\])/igm, plain: "+$2+", xhtml: "<span style='text-decoration: underline;'>$2</span>" },
+		{ plain: '&lt;u&gt;underline&lt;/u&gt;', regex: /(\&lt;u\&gt;(.*?)\&lt;\/u\&gt;)/igm, plain: "+$2+", xhtml: "<span style='text-decoration: underline;'>$2</span>" },
+		{ plain: '&lt;ins&gt;underline&lt;/ins&gt;', regex: /(\&lt;ins\&gt;(.*?)\&lt;\/ins\&gt;)/igm, plain: "+$2+", xhtml: "<span style='text-decoration: underline;'>$2</span>" }
+		]
```
Comment (mweibel) [inline emphasis/candy.js]: Minor nitpick: Indent wrong & semicolon missing ;)

**Comment by melissanoelle** [comment]
This looks good now? :)

**Comment by mweibel** [comment]
You got push access to this repo now.
Remember the rules:
- Not your own plugins should *always* be adapted through PRs
- Own plugins can be adapted as you wish, although also there are PRs preferred

**Pull request merged**